sys/fido2/ctap/ctap_mem.c

```diff
@@ -28,16 +28,22 @@
 #define ENABLE_DEBUG    (0)
 #include "debug.h"
 
+#ifdef BOARD_NATIVE
+#include "mtd_default.h"
+// native mtd is file backed => Start address of flash is 0.
```

Comment:
This line is only needed when running the fido2 layer natively (for testing). In this case I set the `mtd_default` because that is the only one that works correctly natively if I remember correctly.